src/server/utils/authorization.js

```diff
@@ -79,31 +40,22 @@ export const getUserOrgDoc = (userId, orgId = '') => {
     .run();
 };
 
-export const isBillingLeader = (userOrgDoc) => {
+export const isOrgBillingLeader = (userOrgDoc) => {
   return (userOrgDoc && userOrgDoc.role === BILLING_LEADER);
 };
 
-export const requireOrgLeader = (userOrgDoc) => {
-  const legit = isBillingLeader(userOrgDoc);
-  if (!legit) {
-    throw new Error('Unauthorized. User is not a Billing Leader for that organization');
-  }
-};
-
-export const requireOrgLeaderOfUser = async (authToken, userId) => {
+export const isOrgLeaderOfUser = async (authToken, userId) => {
   const r = getRethink();
-  const isLeaderOfUser = await r.table('User')
+  return r.table('User')
```

Comment:
Look at you optimizing

Reply:
man, i know i shouldn't during a refactor, but if not now, when?